src/Controller/TimesheetControllerTrait.php

```diff
@@ -184,6 +190,18 @@ protected function create(Request $request, $redirectRoute, $renderTemplate)
         ]);
     }
 
+    /**
+     * @param User $user
+     * @throws \App\Repository\RepositoryException
+     * @throws \Doctrine\ORM\ORMException
+     * @throws \Doctrine\ORM\OptimisticLockException
+     */
+    protected function stopActiveEntries(User $user)
+    {
+        $activeRecordsLimit = $this->getParameter('kimai.timesheet.active_entries.hard_limit');
```

Comment:
this should to be injected instead of pulled from the container.
```php
if ($limit > 1) { 
    throw new \Exception('Too many entries active'); 
}
```

Reply:
I don't think that this is the right place here. `stopActiveEntries()` is called on every `createAction()` and `startAction()` so this always resolves in an error message.